config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
+  root to: "routines#index"
```

Comment:
This is a small nitpick since I know that you are just getting started. I am able to run the current iteration of your app & your routines#index root route works. However, I would consider adding `before_action :authenticate_user!` to your application controller to force sign-in

Reply:
I will add that in and comment it out to make sure its added later! Thank you for reviewing my models and associations!